.env.sample

```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_FIREBASE_PUBLIC_API_KEY = 
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN = 
+NEXT_PUBLIC_FIREBASE_PROJECT_ID = 
+NEXT_PUBLIC_FIREBASE_DATABASE_URL = 
+NEXT_PUBLIC_FIREBASE_MESSAGINIG_SENDER_ID = 
```

Comment:
Can you rebase with develop & fix this typo? I think the build should work after that

Reply:
I dont have to rebase with develop since the env.sample isn't in develop branch yet.

Reply:
Nah, you do. I made workflow changes

Reply:
Wait, I lied ig, github automatically updates that nvm